modules/reserved-ips/variables.tf

```diff
@@ -0,0 +1,126 @@
+##############################################################################
+# VPC Variables
+##############################################################################
+
+variable "prefix" {
+  description = "The prefix that you would like to append to your resources. Value is only used if no value is passed for the `vpe_name` option in the `cloud_services` input variable."
+  type        = string
+  default     = "vpe"
+}
+
+variable "vpc_name" {
+  description = "Name of the VPC where the Endpoint Gateways will be created. This value is used to dynamically generate VPE names. Value is only used if no value is passed for the `vpe_name` option in the `cloud_services` input variable."
+  type        = string
+  default     = "vpc"
+}
+
+##############################################################################
+# SUBNET Variables
+##############################################################################
+
+variable "subnet_zone_list" {
+  description = "List of subnets in the VPC where gateways and reserved IPs will be provisioned. This value is intended to use the `subnet_zone_list` output from the Landing Zone VPC Subnet Module (https://github.com/terraform-ibm-modules/terraform-ibm-landing-zone-vpc) or from templates using that module for subnet creation."
+  type = list(
+    object({
```

Comment:
Do we really need to ask for all of those informations?

Reply:
`id`, `name` and `zone` are all used in the naming convention in the main module which this replicates. i can omit `cidr`

Reply:
I could remove `cidr`, need others for naming convention.